sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -90,6 +90,7 @@ public static void init() {
    * @throws NoSuchMethodException the NoSuchMethodException
    * @throws InvocationTargetException the InvocationTargetException
    */
+  @SuppressWarnings("LambdaLast")
```

Comment:
fixing all of those would be great, so Kotlin could use the last argument with a lambda if its an interface function, but would be a breaking change

Reply:
this is called trailing lambda btw